components/utils/src/site.rs

```diff
@@ -14,12 +14,15 @@ pub fn get_reading_analytics(content: &str) -> (usize, usize) {
     (word_count, ((word_count + 199) / 200))
 }
 
+/// Result of a successful resolution of an internal link.
 #[derive(Debug, PartialEq, Clone)]
 pub struct ResolvedInternalLink {
+    /// Resolved link target, as absolute URL address.
     pub permalink: String,
-    // The 2 fields below are only set when there is an anchor
-    // as we will need that to check if it exists after the markdown rendering is done
-    pub md_path: Option<String>,
+    /// Internal path to the .md file, without the leading `@/`.
```

Comment:
is that path normalised or not?

Reply:
I don't know what is the definition of "normalised" here, so I'm not sure. The logic to compute this value is the same as before (minus the `Option` wrapper), I only added a docstring to the best of my understanding.
If you have a specific suggestion here to make it clearer, I'll be happy to update.

Reply:
That's fine I got confused whether it was the on disk path or the in md path!